specs/lightclient/beacon-chain.md

```diff
@@ -22,36 +24,66 @@
 - [Helper functions](#helper-functions)
   - [`Predicates`](#predicates)
     - [`eth2_fast_aggregate_verify`](#eth2_fast_aggregate_verify)
+  - [Misc](#misc-2)
+    - [`flags_and_numerators`](#flags_and_numerators)
   - [Beacon state accessors](#beacon-state-accessors)
     - [`get_sync_committee_indices`](#get_sync_committee_indices)
     - [`get_sync_committee`](#get_sync_committee)
+    - [`get_base_reward`](#get_base_reward)
+    - [`get_unslashed_participating_indices`](#get_unslashed_participating_indices)
+    - [`get_flag_deltas`](#get_flag_deltas)
+      - [`get_inactivity_penalty_deltas`](#get_inactivity_penalty_deltas)
   - [Block processing](#block-processing)
+    - [New `process_attestation`](#new-process_attestation)
+    - [New `process_deposit`](#new-process_deposit)
     - [Sync committee processing](#sync-committee-processing)
   - [Epoch processing](#epoch-processing)
-    - [Components of attestation deltas](#components-of-attestation-deltas)
+    - [New `process_justification_and_finalization`](#new-process_justification_and_finalization)
+    - [New `process_rewards_and_penalties`](#new-process_rewards_and_penalties)
     - [Final updates](#final-updates)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 <!-- /TOC -->
 
 ## Introduction
 
-This is a standalone beacon chain patch adding light client support via sync committees.
+This is a patch implementing the first hard fork to the beacon chain, tentatively named HF1 pending a permanent name. It has three main features:
+
+* Light client support via sync committees
+* Incentive accounting reforms, reducing spec complexity
+  and [TODO] reducing the cost of processing chains that have very little or zero participation for a long span of epochs
+* Fork choice rule changes to address weaknesses recently discovered in the existing fork choice
 
 ## Constants
 
+### Participation flags
+
 | Name | Value |
-| - | - | 
-| `BASE_REWARDS_PER_EPOCH` | `uint64(5)` |
+| - | - |
+| `TIMELY_HEAD_FLAG` | `0` |
```

Comment:
These are different than the flags in the description? 

```
FLAG_TARGET: correct source and target
FLAG_TIMELY: got included within ~5 slots
FLAG_HEAD_AND_VERY_TIMELY: got the head correct and got included within 1 slot
```

Reply:
Right, https://github.com/ethereum/eth2.0-specs/commit/331b0f6da1c5dfb31035f67aeae4b090a48001bc changed it.